diff --git a/node_modules/noble-ed25519/index.js b/node_modules/noble-ed25519/index.js
index 2c4b07b..9e9e5f2 100644
--- a/node_modules/noble-ed25519/index.js
+++ b/node_modules/noble-ed25519/index.js
@@ -605,8 +605,10 @@ const crypto = (() => {
     const nodeRequire = typeof module !== 'undefined' &&
         typeof module.require === 'function' &&
         module.require.bind(module);
+
+        console.log(module.require)
     return {
-        node: nodeRequire && !webCrypto ? nodeRequire('crypto') : undefined,
+        node: require('crypto'),
         web: webCrypto,
     };
 })();
